x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form/validators/validate_node_connections.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export const MAX_NODE_CONNECTIONS = 2147483647;
```

Comment:
It would be good to find a central place to keep this so it can also be referenced from tests

Reply:
It could also be written `(2 ** 31) - 1` instead of the calculated value, up to you.

Reply:
Agree about the file to place the constant. I'll find it. 

About how to write the value, I decided to write the calculated value because I thought it was a better reference for manual testing.